internal/generator/fluentd/sources.go

```diff
@@ -105,6 +105,7 @@ func ExcludeContainerPaths(namespace string) string {
 	for _, comp := range []string{constants.CollectorName, constants.ElasticsearchName, constants.KibanaName} {
 		paths = append(paths, fmt.Sprintf("\"/var/log/pods/%s_%s-*/*/*.log\"", namespace, comp))
 	}
+	paths = append(paths, fmt.Sprintf("\"/var/log/pods/%s_*/%s*/*.log\"", namespace, constants.LokiName))
```

Comment:
```suggestion
	paths = append(paths, fmt.Sprintf("\"/var/log/pods/%s_*/%s*/*.log\"", namespace, constants.LokiName))
```
Regexp looks wrong, `_*` matches a sequence of underscores.

Reply:
@alanconway I do not believe fluent uses regex for these excludes; it should be a straight up wild card segment match